universal-login-react/src/ui/Onboarding/Onboarding.tsx

```diff
@@ -57,7 +57,8 @@ export const Onboarding = (props: OnboardingProps) => {
       action: 'Wallet creation',
       transactionHash: '0xee9270ccdeb9fcb92b3ec509ba11ba2362ab32ba8f...'}
     );
-    const wallet = await deploy(ensName, gasParameters.gasPrice.toString(), gasParameters.gasToken);
+    const {waitToBeSuccess, deployedWallet: wallet} = await deploy(ensName, gasParameters.gasPrice.toString(), gasParameters.gasToken);
+    await waitToBeSuccess();
```

Comment:
`deployedWallet` should not be returned by `deploy`, but  by `waitToBeSuccess` 

#deployedwalletonsuccess